__test__/components/__snapshots__/ResetFormContainer.test.js.snap

```diff
@@ -0,0 +1,56 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Reset Form should match snapshot 1`] = `
+<div
+  className="signup-form-container"
+>
+  <h3
+    className="text-center mt-5"
+  >
+    <img
+      alt="facebook logo"
+      className="signup-title"
+      src="test-file-stub"
+    />
+    <hr />
+  </h3>
+  <form
+    className="signup-form mb-5"
+    onSubmit={[Function]}
+  >
+    <div
+      className="input-field"
+    >
+      <label
+        htmlFor="email"
+      />
+      <input
+        className="sc-bdVaJa hTWEyw"
+        id="email"
+        name="email"
+        onChange={[Function]}
+        placeholder="Email"
+        required={true}
+        type="email"
+        value=""
+      />
+      <small
+        className="text-danger ml-3 form-text"
+      />
+    </div>
+    <p
+      className="text-center"
+    >
+      A reset link would be sent to your email
+    </p>
+    <button
+      className="sc-htpNat itNkkK"
```

Comment:
This classname is coming in because it is a styled component. For tests that have this, you want to do import 'jest-styled-components'; in the test to prevent that classname regeneration issue.